2021-08-13 13:39:49,993 INFO    MainThread:3259 [wandb_setup.py:_flush():69] setting env: {}
2021-08-13 13:39:49,994 WARNING MainThread:3259 [wandb_setup.py:_flush():69] could not save program above cwd: /cs/casmip/nirm/embryo_project_version1/embyo_projects_codes/lord-pytorch-unet/lord_exp.py
2021-08-13 13:39:49,994 INFO    MainThread:3259 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-13 13:39:49,994 INFO    MainThread:3259 [wandb_init.py:_log_setup():337] Logging user logs to /cs/labs/josko/nirm/embryo_project_version1/embyo_projects_codes/lord-pytorch-unet/wandb/run-20210813_133949-3esqtjbd/logs/debug.log
2021-08-13 13:39:49,994 INFO    MainThread:3259 [wandb_init.py:_log_setup():338] Logging internal logs to /cs/labs/josko/nirm/embryo_project_version1/embyo_projects_codes/lord-pytorch-unet/wandb/run-20210813_133949-3esqtjbd/logs/debug-internal.log
2021-08-13 13:39:49,994 INFO    MainThread:3259 [wandb_init.py:init():370] calling init triggers
2021-08-13 13:39:49,994 INFO    MainThread:3259 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-08-13 13:39:49,994 INFO    MainThread:3259 [wandb_init.py:init():419] starting backend
2021-08-13 13:39:49,994 INFO    MainThread:3259 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-13 13:39:50,001 INFO    MainThread:3259 [backend.py:ensure_launched():135] starting backend process...
2021-08-13 13:39:50,007 INFO    MainThread:3259 [backend.py:ensure_launched():140] started backend process with pid: 3326
2021-08-13 13:39:50,008 INFO    MainThread:3259 [wandb_init.py:init():424] backend started and connected
2021-08-13 13:39:50,009 INFO    MainThread:3259 [wandb_init.py:init():472] updated telemetry
2021-08-13 13:39:50,009 INFO    MainThread:3259 [wandb_init.py:init():491] communicating current version
2021-08-13 13:39:50,789 INFO    MainThread:3259 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.12.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-13 13:39:50,789 INFO    MainThread:3259 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-08-13 13:39:51,142 INFO    MainThread:3259 [wandb_init.py:init():529] starting run threads in backend
2021-08-13 13:39:51,210 INFO    MainThread:3259 [wandb_run.py:_console_start():1623] atexit reg
2021-08-13 13:39:51,213 INFO    MainThread:3259 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.REDIRECT
2021-08-13 13:39:51,213 INFO    MainThread:3259 [wandb_run.py:_redirect():1502] Redirecting console.
2021-08-13 13:39:51,214 INFO    MainThread:3259 [wandb_run.py:_redirect():1558] Redirects installed.
2021-08-13 13:39:51,214 INFO    MainThread:3259 [wandb_init.py:init():554] run started, returning control to user process
2021-08-13 13:40:05,721 INFO    MainThread:3259 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 255
2021-08-13 13:40:05,721 INFO    MainThread:3259 [wandb_run.py:_restore():1565] restore
2021-08-13 13:40:08,378 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1010
  total_bytes: 1010
}

2021-08-13 13:40:08,786 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1010
  total_bytes: 1010
}

2021-08-13 13:40:09,167 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1010
  total_bytes: 13060
}

2021-08-13 13:40:09,268 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1010
  total_bytes: 13062
}

2021-08-13 13:40:09,369 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1010
  total_bytes: 13062
}

2021-08-13 13:40:09,470 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1010
  total_bytes: 13062
}

2021-08-13 13:40:09,571 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1010
  total_bytes: 13062
}

2021-08-13 13:40:09,672 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13062
  total_bytes: 13062
}

2021-08-13 13:40:09,773 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13062
  total_bytes: 13062
}

2021-08-13 13:40:09,873 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13062
  total_bytes: 13062
}

2021-08-13 13:40:09,974 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13062
  total_bytes: 13062
}

2021-08-13 13:40:10,075 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13062
  total_bytes: 13062
}

2021-08-13 13:40:10,176 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13062
  total_bytes: 13062
}

2021-08-13 13:40:10,603 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13062
  total_bytes: 13062
}

2021-08-13 13:40:10,704 INFO    MainThread:3259 [wandb_run.py:_wait_for_finish():1715] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13062
  total_bytes: 13062
}

2021-08-13 13:40:11,751 INFO    MainThread:3259 [wandb_run.py:_show_files():1937] logging synced files
